tests/Zend/XmlRpc/ServerTest.php

```diff
@@ -783,9 +783,4 @@ class Zend_XmlRpc_Server_testResponse extends Zend_XmlRpc_Response
 
 class Zend_XmlRpc_Server_testRequest extends Zend_XmlRpc_Request
 {
-}
-
-// Call Zend_XmlRpc_ServerTest::main() if this source file is executed directly.
-if (PHPUnit_MAIN_METHOD == "Zend_XmlRpc_ServerTest::main") {
-    Zend_XmlRpc_ServerTest::main();
-}
```

Comment:
why is this (and other similar changes in tests/*) removed here? please match commit messages with the changes.

Reply:
Mr @glensc I break this commit into 2 commits:
https://github.com/Shardj/zf1-future/pull/291/commits/8a6f8b1ac572e05cd29151d7b990b27988af38c6
https://github.com/Shardj/zf1-future/pull/291/commits/cbf41303cc4ad76697cc5df07764178803daebbc